### PR TITLE
DNC 3.1.0 Preview3

### PR DESCRIPTION
**Pull request by MarkStega**

### CODE REVIEW

**Comment by MarkStega** [comment]
This PR now supports APS.Net Core 3.1.0 Preview 3

**Comment by MarkStega** [comment]
I made the changes. My starting point was the master branch before you made all of the build changes. Everything appears to build now.

**Comment by MarkStega** [comment]
I attempted an update to the released DNC 3.1.0; Everything builds properly but there is some JS interop error at runtime, I do not understand the implementation well enough to successfully debug. I'll keep trying to read the code but could use some help!

**Comment by MarkStega** [comment]
I made changes for the released DNC 3.1.0 however there is now a jsinterop failure at runtime and I am not getting logs. I'll continue to try to understand but could use some help! I'll write more detail in a PM on gitter.

**Comment by chucker** [comment]
> I made changes for the released DNC 3.1.0 however there is now a jsinterop failure at runtime and I am not getting logs. I'll continue to try to understand but could use some help! I'll write more detail in a PM on gitter.

Is this https://github.com/aspnet/AspNetCore/issues/17535? It sounds like you only need to restart VS to fix it.

**Comment by MarkStega** [comment]
@chucker Did you remember to put the explicit script in your index.html? You need 
```
    <script src="_content/Blazor.Extensions.Logging/blazor.extensions.logging.js"></script>
```

File: test/Blazor.Extensions.Logging.Test/Blazor.Extensions.Logging.Test.csproj [hunk 1]
```diff
@@ -7,10 +7,10 @@
...
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Blazor" Version="3.1.0-preview4.19579.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="3.1.0-preview4.19579.2" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.AspNetCore.Blazor.DevServer" Version="3.1.0-preview4.19579.2" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.AspNetCore.Blazor.HttpClient" Version="3.1.0-preview4.19579.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor" Version="3.2.0-preview1.20073.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="3.2.0-preview1.20073.1" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor.DevServer" Version="3.2.0-preview1.20073.1" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor.HttpClient" Version="3.2.0-preview1.20073.1" />
```
Comment (chucker) [inline test/Blazor.Extensions.Logging.Test/Blazor.Extensions.Logging.Test.csproj]: You should change the package references here; some of the names have changed (which is why you're not getting `preview2` packages).

https://devblogs.microsoft.com/aspnet/blazor-webassembly-3-2-0-preview-2-release-now-available/

> Update your package references and namespaces as described below:
Microsoft.AspNetCore.Blazor -> Microsoft.AspNetCore.Components.WebAssembly
Microsoft.AspNetCore.Blazor.Build -> Microsoft.AspNetCore.Components.WebAssembly.Build
Microsoft.AspNetCore.Blazor.DevServer -> Microsoft.AspNetCore.Components.WebAssembly.DevServer
Microsoft.AspNetCore.Blazor.Server -> Microsoft.AspNetCore.Components.WebAssembly.Server
Microsoft.AspNetCore.Blazor.HttpClient -> Microsoft.AspNetCore.Blazor.HttpClient (unchanged)
Microsoft.AspNetCore.Blazor.DataAnnotations.Validation -> Microsoft.AspNetCore.Components.DataAnnotations.Validation
Microsoft.AspNetCore.Blazor.Mono -> Microsoft.AspNetCore.Components.WebAssembly.Runtime
Mono.WebAssembly.Interop -> Microsoft.JSInterop.WebAssembly

**Comment by attilah** [comment]
Closing as package is update to Blazor RC1, available on [Nuget](https://www.nuget.org/packages/Blazor.Extensions.Logging/3.1.0-rc1-020501)

**Pull request closed without merging**